gnd/src/main/java/com/google/android/gnd/ui/editrecord/EditRecordViewModel.java

```diff
@@ -48,13 +48,19 @@
 import com.google.common.collect.ImmutableList;
 import java.util.Arrays;
 import java.util.Collections;
+
+import io.reactivex.BackpressureStrategy;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+import io.reactivex.processors.BehaviorProcessor;
 import java8.util.Optional;
 import java8.util.stream.Stream;
 import javax.inject.Inject;
 
 // TODO: Save draft to local db on each change.
 public class EditRecordViewModel extends AbstractViewModel {
   private static final String TAG = EditRecordViewModel.class.getSimpleName();
+  private static final String NEW_RECORD_ID_ARG_PLACEHOLDER = "NEW_RECORD";
```

Comment:
Even though it's no longer referenced directly inside the Fragment, this constant refers to a key used directly in the FragmentArgs, which are generated for the Fragment and not the ViewModel, so it would probably be more correct to put it there.